### ISSUE TITLE
Hiragana and katakana

### ISSUE THREAD

**Issue by ailectra** [infinyte7/Write-Kanji#2]
Hi,

Just wanted to let you know that I converted animCJK kana data into MakeMeAHanzi compatible data : https://github.com/ailectra/kana-json

I have no trouble when writing hiragana with overlapping stroke in Hanzi Writer.

Maybe you could fetch this dataset when a kana is used?

**Comment by krmanik** [comment]
This is great. Thanks for creating this.
I have tried it works fine. I recommend you to create pull request to hanzi-writer-data-jp. It really nice. I will create word list for Japanese language using this. 
I have tried to implement earlier but didn't work. So I am curious to know how you have created this? It is working without modifying existing hanzi-writer.js code.

**Comment by ailectra** [comment]
As coordinates are not the same in animCJK and MakeMeAHanzi, I had to transform the coordinates once I exported the SVG into JSON format. The x coordinate is the same but you have to use the following formula for y 
`
y = 900 - y
`

For example, if you have a point `(660,211)`, it becomes `(660, 900-211)=(660, 689)`

**Comment by Ross-Da-Boss** [comment]
@infinyte7 if you could add kana support, it'd be great.

**Comment by krmanik** [comment]
Hi, In next update I will add the kana support. Stay tuned.

**Comment by krmanik** [comment]
- [View this to add writing component to exiting deck](https://github.com/infinyte7/Write-Kanji/tree/master/Add%20to%20Existing%20Deck%20(Kanji)/version%202.0)
- [Watch demo video to add writing component to exisitng deck](https://youtu.be/_dZeREl1TI8)

**Comment by krmanik** [comment]
Hi @ailectra,
Can you explain how you have created the strokes data for loop? I have checked the data of `あ` and it is different from KanjiVG so how the data is generated? Any ideas will be helpful and generating these characters, [caoshu-hanzi-writer-data](https://github.com/krmanik/caoshu-hanzi-writer-data)

<img height="450px" src="https://user-images.githubusercontent.com/12841290/192339156-b933d13e-6df3-46e3-b202-bb9fda3d9f1e.png"></img>

**Comment by ailectra** [comment]
Hi,

I used the data from [animCJK](https://github.com/parsimonhi/animCJK).

**Comment by krmanik** [comment]
The data for third and fourth strokes are merged and all strokes are flipped with changed coordinates. So, you have just put the two strokes data into one?
[Left - animCJK] | [Right - kana json]
![image](https://user-images.githubusercontent.com/12841290/192480232-19135da0-bbb0-4cd0-b1d8-633b7053146e.png)

**Comment by ailectra** [comment]
Yes that's exactly what I did. I flipped the coordinates because that's how hanzi-writer handles them. As for the merged strokes, it's because hanzi-writer doesn't handle overlapping strokes. It's the only way to make it work.

**Comment by krmanik** [comment]
That's great
